com.wireframesketcher.WireframeSketcher.json

```diff
@@ -0,0 +1,81 @@
+{
+  "id": "com.wireframesketcher.WireframeSketcher",
+  "runtime": "org.gnome.Platform",
+  "runtime-version": "46",
+  "sdk": "org.gnome.Sdk",
+  "sdk-extensions": [],
+  "command": "WireframeSketcher",
+  "separate-locales": false,
+  "tags": [
+    "proprietary"
+  ],
+  "rename-appdata-file": "wireframesketcherstudio.appdata.xml",
+  "rename-desktop-file": "wireframesketcherstudio.desktop",
+  "rename-icon": "WireframeSketcher",
+  "finish-args": [
+    "--require-version=1.7.1",
```

Comment:
why do you need specifically this version?

Reply:
Not sure. This is what Eclipse app does.